README.md

```diff
@@ -28,6 +28,43 @@ gulp safari
 
 Find the extension folder in `tmp/safari`. Follow Apple instructions to install the extension to your browser.
 
+### Install on Safari (build in a docker container)
+
+If you don't have the necessary software (such as the correct version of `node` or `gulp`) to build the extension, you can build within a docker container (requires [docker](https://docs.docker.com/docker-for-mac/install/)).
+
+First, run interactive bash in a container that has all the required tools:
+
+```bash
+docker run -it --name octotree-build gabrielaraujof/nodejs-gulp-bower bash
+```
+
+Within the container:
+
+```bash
+git clone https://github.com/ovity/octotree.git
```

Comment:
I suggest adding `--depth 1`, so cloning happens faster, and we do not need the whole history of the repository